src/workloads/execution/2DIndex.yml

```diff
@@ -0,0 +1,40 @@
+# This workload inserts ~1GB of documents in the first phase, and creates a 2d index on them in the second.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
```

Comment:
FYI, you can define phases in a separate YAML file to avoid repetition. E.g. for this phase, instead of 

```
  - Repeat: 1
    Database: &db test
    Threads: 1
    CollectionCount: 1
    DocumentCount: 500000
    BatchSize: 10000
    Document:
      a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
```

you can write:

```
   - ExternalPhaseConfig:
      Path: ../../phases/execution/IndexWorkloadPhases.yml
      Key: InsertPhase
      Parameters:
        MyDocument: {a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]}
```

Then create a `src/phases/execution/IndexWorkloadPhases.yml` that looks like:

```
InsertPhase:
  Repeat: 1
  Database: &db test
  Threads: 1
  CollectionCount: 1
  DocumentCount: 500000
  BatchSize: 10000
  Document: {^Parameter: {Name: "MyDocument", Default: {}}}
```

The YAML structure in the external phase config will replace the YAML structure starting with `ExternalPhaseConfig` in the workload file. You can also change the `MyDocument` field in `Parameters` to insert different documents for different workloads.